aten/src/ATen/native/ConvolutionMM2d.cpp

```diff
@@ -369,6 +341,11 @@ static void slow_conv2d_backward_parameters_out_cpu_template(
   auto input = input_.contiguous();
   auto grad_output = grad_output_.contiguous();
 
+  if (grad_bias.defined()) {
+    Tensor grad_bias_mut = grad_bias;
+    at::sum_out(grad_bias_mut, grad_output, IntArrayRef{0, 2, 3});
+  }
+
```

Comment:
This is a much nicer way to compute `grad_bias` and seems correct AFAICT.

Couple questions regarding this:
* Are there any tests verifying the `grad_bias` calculation? I briefly checked but nothing stood out
* Do you by chance have any comparative benchmarking results for the change?

Reply:
>Are there any tests verifying the grad_bias calculation? I briefly checked but nothing stood out

The nn tests are a bit confusing, but I think the gradcheck is done here:
https://github.com/pytorch/pytorch/blob/59fcbd172b5dfdeb12e2f1b7a78c2ce95c1eb680/torch/testing/_internal/common_nn.py#L5807

That test includes inputs and parameters, so `bias` should be covered.


>Do you by chance have any comparative benchmarking results for the change?

I've done some simple benchmarking which showed positive results. LMK if you want more. 

For CUDA, I've run a convolution in a loop under nvprof with these parameters:
```
input = torch.rand(1, 50, 260, 260, device=device, requires_grad=False)
weight = torch.rand(10, 50, 3, 3, device=device, requires_grad=False)
bias = torch.rand(10, device=device, requires_grad=True)
```
The result is it calls one kernel instead of two (cublas gemm here becomes `dot_kernel` + `reduce_1Block_kernel`). The kernel execution time is also noticeably faster:

|          | Master (us) | This PR (us) |
|----------------:|:-----------:|:--------------------:|
|  Forward |     23.8    |         14.6         |
| Backward |     22.4    |         18.1         |

On CPU without nvprof, the results are dwarfed by the main convolution and the same shapes showed no measurable difference.  So I also tried a smaller input size with H=120, C_in = 2 and that showed some minor improvement in the forward case (presumably from vectorization).

|                  | Master (us) | This PR (us) |
|-----------------:|:-----------:|:--------------------:|
|    Forward (Big) |    15,100   |        15,100        |
|   Backward (Big) |    13,100   |        13,100        |
|  Forward (Small) |     526     |          499         |
| Backward (Small) |    1,050    |         1,050        |